tasks/browserify.js

```diff
@@ -29,6 +30,11 @@ module.exports = function (gulp, plugins, options) {
       ignore: ['jquery']
     });
 
+    // The ignore is for the jQuery stream below
+    bundler.transform(babelify.configure({
+      ignore: /_stream_0\.js/
```

Comment:
Didn't know that existed, I'll do that tomorrow. Also need to switch to ESLint—JSHint doesn't support ES6 in the way that we need.